enarx-keep/src/main.rs

```diff
@@ -29,6 +30,9 @@ struct Exec {
     #[structopt(short, long)]
     keep: Option<String>,
 
+    #[structopt(short, long)]
+    shim: Option<String>,
```

Comment:
I'm wondering if this can be simplified with:
```suggestion
    #[structopt(parse(from_os_str))]
    shim: Option<PathBuf>,
```

Reply:
perfect... changed